cea/globalvar.py

```diff
@@ -77,7 +77,7 @@ def __init__(self):
         # Commands for the evolutionary algorithm
 
 
-        self.initialInd = 2  # number of initial individuals
+        self.initialInd = 5  # number of initial individuals
```

Comment:
ok. i'll move it to the configuration file once you're done. just give me a heads up when you're ready.